dev/doc/critical-bugs

```diff
@@ -346,11 +439,12 @@ Conversion machines
   component: "virtual machine" (compilation to bytecode ran by a C-interpreter)
   summary: buffer overflow on large records and closures
   introduced: 8.1
-  impacted released versions: 8.1-now
+  impacted released versions: 8.1-8.13
   impacted coqchk versions: none (no virtual machine in coqchk)
-  fixed in:
+  fixed in: v8.13 (#13431)
```

Comment:
This whole entry should be removed, as it is a duplicate of the very first entry. This specific mention of #13431 could be applied to the previous entry.